### PR TITLE
[feature2] Second feature for this project

### PR DESCRIPTION
**Pull request by mike-bold**
Just added new feature:

1. New folder created
2. New file was added into the folder

### CODE REVIEW

File: feature2/another-feature.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+feature2 branch 
+
+Another line added
```
Comment (lukiluk5) [inline feature2/another-feature.txt:3]: It was a good idea to add this line Mike!

File: feature2/another-feature.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+feature2 branch 
+
```
Comment (lukiluk5) [inline feature2/another-feature.txt:2]: Is this space necessary?

**Review by lukiluk5 — COMMENTED** [review]
End of finish review
Still not approve

File: feature2/another-feature.txt (hunk 2 above)
Comment (mike-bold) [reply feature2/another-feature.txt:2]: Yes this space is necessary for readability reason.

**Pull request merged**